go.sum

```diff
@@ -1,3 +1,5 @@
+github.com/AdamKorcz/go-fuzz-headers v0.0.0-20210319161527-f761c2329661 h1:LxxqfxscKXL1kv7QNh4nggNf4Ais8B0ME8zWMCAsttY=
```

Comment:
Can this package live in some org repo, rather than under your personal account?